composer.json

```diff
@@ -18,10 +18,10 @@
     "require": {
         "php": ">=7.2",
         "twilio/sdk": "~5.16",
-        "illuminate/notifications": "^5.5 || ^6.0 || ^7.0",
-        "illuminate/support": "^5.5 || ^6.0 || ^7.0",
-        "illuminate/events": "^5.5 || ^6.0 || ^7.0",
-        "illuminate/queue": "^5.5 || ^6.0 || ^7.0"
```

Comment:
5.5 is still in security support & deferred providers seem to be present in 5.5.
Is there a reason why you're dropping support?

Reply:
` Illuminate\Contracts\Support\DeferrableProvider` has been introduced at 5.8